polkit/validate/validate.go

```diff
@@ -283,3 +285,41 @@ func validateDefaultAuth(auth []Element, name string) error {
 	}
 	return nil
 }
+
+var ruleNameSuffixRegexp = regexp.MustCompile(`^([\w-]+)(\.[\w-]+)*$`)
```

Comment:
rule out `---|--.--|foo.---|foo.` and similar
```suggestion
var ruleNameSuffixRegexp = regexp.MustCompile(`^(\w[\w-]*)(\.\w[\w-]*)*$`)
```

Reply:
updated, thanks!